airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/buffers/MemoryBoundedLinkedBlockingQueue.java

```diff
@@ -80,6 +81,16 @@ public MemoryBoundedLinkedBlockingQueue.MemoryItem<E> poll() {
     return null;
   }
 
+  @Override
```

Comment:
Does it make sense to merge into [this branch](https://github.com/airbytehq/airbyte/pull/26375) and have it all merged as one since both branches update this method

Reply:
i'm ok either way. i'll merge that branch in right after